### PR TITLE
Added clarity to rules (per CentralCommand Mike)

### PR DESCRIPTION
**Pull request by fortepc**
CentralCommand's comments on the forums added important nuance to this rule that is important to integrate into this post

>4)  If you go ask ChatGPT, review the response, test it, correct the flaws and then post a working solution - you’ll probably get a solution checkmark and be celebrated as a great member of the community
source: https://community.home-assistant.io/t/want-to-help-others-leave-your-ai-at-the-door/523704/108
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by fortepc — COMMENTED** [review]
Corrected a slight typo

File: source/_posts/2023-01-23-help-others-leave-ai-at-the-dor.markdown [hunk 1]
```diff
@@ -9,12 +9,14 @@ comments: true
 categories: Public-Service-Announcement
 ---
 
-Today we're introducing a new rule for the Home Assistant community: it's no longer allowed to use ChatGPT or other AI systems to help others.
+Today we're introducing a new rule for the Home Assistant community: it's no longer allowed to use ChatGPT or other AI systems to provide untested or unsourced solutions to help others.
```
Comment (frenck) [inline source/_posts/2023-01-23-help-others-leave-ai-at-the-dor.markdown:12]: This wording change isn't needed and is not the takeaway.

File: source/_posts/2023-01-23-help-others-leave-ai-at-the-dor.markdown [hunk 2]
```diff
@@ -9,12 +9,14 @@ comments: true
...
 
-Although these systems generate elaborate and well structured answers, they are wrong. Often they are wrong in subtle ways which only someone with the right expertise could detect. And those people wouldn't need AI to have written that answer.
+Although these systems generate elaborate and well structured answers, often they are wrong in subtle ways which only someone with the right expertise could detect. And those people wouldn't need AI to have written that answer.
```
Comment (frenck) [inline source/_posts/2023-01-23-help-others-leave-ai-at-the-dor.markdown:14]: Not often. They are wrong.

File: source/_posts/2023-01-23-help-others-leave-ai-at-the-dor.markdown [hunk 3]
```diff
@@ -9,12 +9,14 @@ comments: true
...
 
 We appreciate that people want to help others, but if you don't have the knowledge, leave it to someone else. Giving an incorrect answer makes things worse. You are wasting everybody's time, including the person asking the question. Trying out an answer that doesn't work is frustrating because you think you're doing something wrong.
 
-Using an AI system to generate an answer while not providing attribution to AI will result in a ban. If you use attribution, we will delete your post and issue a warning. This also means suggesting someone "ask ChatGPT" is not an acceptable response.
+Using an AI system to generate your answer without proper research or testing is strictly banned. Doing so with attribution will result in your post deleted and a warning issued. If you post that same answer without any attribution this will result in a ban. This also means suggesting someone "ask ChatGPT" is not an acceptable response.
```
Comment (frenck) [inline source/_posts/2023-01-23-help-others-leave-ai-at-the-dor.markdown:18]: This is literally the same as it already was.

TL;DR: Don't use it. Don't care if it is researched or not. We can't see, nor do we accept. If we detect you are using ChatGPT you will risk a ban. That is the whole point of this blog.

If you are looking to bending rules or to find edge cases: I'm sorry.

File: source/_posts/2023-01-23-help-others-leave-ai-at-the-dor.markdown [hunk 4]
```diff
@@ -9,12 +9,14 @@ comments: true
...
+
+Using an AI system as **part** of your research while helping answer a question is fine, so long as your provided solution is actually properly tested. In this case attribution will not result in any punitive action.
```
Comment (frenck) [inline source/_posts/2023-01-23-help-others-leave-ai-at-the-dor.markdown:20]: > In this case attribution will not result in any punitive action.

That is actually incorrect. As per blog post and original discussion:

> Doing so with attribution will result in your post being deleted and a warning issued.

Attribution will lead to deletion with a warning, which may result in banning (in case multiple warnings have been issued as per normal handling).

../Frenck

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
Closing this PR, as a duplicate of others too.

The TL;DR is really simple: Don't use ChatGPTs output with helping people. Doing so, may results into bans. The blog post couldn't be any clearer in the first paragraph:

> it's no longer allowed to use ChatGPT or other AI systems to help others.

It's that easy.

../Frenck
